pkg/handler/types.go

```diff
@@ -1,18 +1,22 @@
-package runner
+package handler
+
+import (
+	"github.com/water-hole/ansible-operator/pkg/runner"
+)
 
 const (
 	host = "localhost"
 )
 
 type Status struct {
-	Ok               int       `json:"ok"`
-	Changed          int       `json:"changed"`
-	Skipped          int       `json:"skipped"`
-	Failures         int       `json:"failures"`
-	TimeOfCompletion EventTime `json:"completion"`
+	Ok               int              `json:"ok"`
+	Changed          int              `json:"changed"`
+	Skipped          int              `json:"skipped"`
+	Failures         int              `json:"failures"`
+	TimeOfCompletion runner.EventTime `json:"completion"`
```

Comment:
`EventTime` is the one part of this that doesn't quite fit on either side. It's just an implementation detail, as a result of `time.Time` not being natively supported for JSON serialization. Maybe it should live in a `util` package or similar? Thoughts?

Reply:
> result of time.Time not being natively supported for JSON serialization

`time.Time` does support serialization it just didn't deserialize and serialize by default as the same format as ansible-runner. 

I don't love a package named `util` or `utils`. If we wanted `runner/time` package so that the import is `water-hole/ansible-operator/pkg/runner/time` and it can be used `time.Event` or `time.Time` or `time.EventTime`. I like this approach because the import tells you what it is and what it does. Thoughts?

Reply:
Leaving it in `runner/` generally seems reasonable if the custom serialization is there to match what ansible does. Although I'm not sure it's worth making a package for just this. Do you think putting it in `runner/time` is better than leaving it in `runner/`? Either seems fine to me.